website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -3554,7 +3554,7 @@ Now, consider the same pipeline, but using a windowing function:
 
 **Figure 5:** `GroupByKey` and `ParDo` with windowing, on a bounded collection.
 
-As before, the pipeline creates a bounded `PCollection` of key/value pairs. We
+As before, the pipeline creates a bounded `PCollection` of file lines. We
```

Comment:
I'd replace "of file lines" with "by reading lines from a file using `TextIO`." I think how you have it earlier is clear.